tests/scripts/thread-cert/pktverify/packet_filter.py

```diff
@@ -572,11 +572,17 @@ def filter_LLABMA(self, **kwargs):
     def filter_LLARMA(self, **kwargs):
         return self.filter(lambda p: p.ipv6.dst == consts.LINK_LOCAL_ALL_ROUTERS_MULTICAST_ADDRESS, **kwargs)
 
-    def filter_AMPLFMA(self, mpl_seed_id: int = None, **kwargs):
+    def filter_AMPLFMA(self, mpl_seed_id: Union[int, Tuple[Ipv6Addr, int]] = None, **kwargs):
```

Comment:
```suggestion
    def filter_AMPLFMA(self, mpl_seed_id: Union[int, Ipv6Addr] = None, **kwargs):
```
Since `RLOC16` is the last 2 bytes of `RLOC` address, it's possible to calculate `RLOC16` directly from `RLOC`:

```
import struct

rloc16 = struct.unpack('>H', rloc[-2:])
```